src/cryptoadvance/specter/templates/node/node_settings.jinja

```diff
@@ -208,5 +220,24 @@
 			}
 		});
 	}
+
+    function toggleExplainer() {
+        let toggle = document.getElementById('toggle-explainer');
+        let explainerText = document.getElementById('text-explainer');
+        if (explainerText.classList.contains('hidden')) {
+            explainerText.classList.remove("hidden");
+            explainerText.classList.add("flex-center")
+            explainerText.classList.add("flex-column")
```

Comment:
Add these classes in the html directly, instead of adding it here through JavaScript. This is because "else" part is not handling removal of these two classes, and "hidden" class is anyways going to hide the element. Thus there would be no side effect due to presence of these classes with the hidden class.

Reply:
Makes sense, changed that. I was already thinking in the back of my mind to make the explainer a web component and keep it thus very flexible and put more in there and less in the html. But, we can tackle this at later stage if we it is worth it to make it a separate web component.